src/register-subscriptions.ts

```diff
@@ -0,0 +1,149 @@
+import { ethers } from 'ethers';
+import { NonceManager } from '@ethersproject/experimental';
```

Comment:
The NounceManager makes it easy to submit transactions in parallel

Reply:
All good, I've used it myself many times :rocket: